doc/news/_preparation_next_release.md

```diff
@@ -1,43 +1,44 @@
-# 0.9.<<VERSION>> Release
+# 0.9.2 Release
 
-This release did not happen yet.
+Not yet released.
 
-Please update this file within PRs accordingly.
-For non-trivial changes, you can choose to be
-part of the highlighted changes. Please make
-sure to add some short tutorial (checked by
-shell recorder) or asciinema for highlighted items.
+- shortDesc: `elektrad` in Go, KDE and GNOME Integration, Bugfixes and Documentation Improvements
 
-Please add your name at the end of every contribution.
-**Syntax:** _(your name)_
-
-<<`scripts/generate-news-entry`>>
-
-We are proud to release Elektra 0.9.<<VERSION>>.
+We are proud to release Elektra 0.9.2.
 
 ## What is Elektra?
 
 Elektra serves as a universal and secure framework to access
 configuration settings in a global, hierarchical key database.
 For more information, visit [https://libelektra.org](https://libelektra.org).
 
-You can also read the news [on our website](https://www.libelektra.org/news/0.9.<<VERSION>>-release)
+You can also read the news [on our website](https://www.libelektra.org/news/0.9.2-release)
 
 ## Highlights
 
-- <<HIGHLIGHT1>>
-- <<HIGHLIGHT2>>
-- <<HIGHLIGHT3>>
+- `elektrad` rewritten in Go
+- KDE and GNOME Integration
+- Bugfixes and Documentation Improvements
+
+### `elektrad` rewritten in Go
+
+[elektrad](https://www.libelektra.org/tools/elektrad) provides an HTTP API to access Elektra remotely. `elektrad` is now completely rewritten in Go, which drastically improves the performance by leveraging the new [go-elektra](https://github.com/ElektraInitiative/go-elektra/) bindings instead of calling the `kdb` command-line tool on every request. The new Elektrad creates a session per user to reuse the same kdb handle for better performance. _(Raphael Gruber)_
 
-### <<HIGHLIGHT1>>
+### KDE and GNOME Integration
 
-### <<HIGHLIGHT2>>
+We are working on integrating Elektra into [KDE](https://kde.org/) and [GNOME](https://www.gnome.org/).
 
-### <<HIGHLIGHT2>>
+A new Elektra plugin called `kconfig` was added which can read and write KDE's kconfig ini files. Additionally we are working on a [fork](https://github.com/ElektraInitiative/kconfig) of KDE's `KConfig` configuration system and patching it to use libelektra. _(Dardan Haxhimustafa)_ and _(Felix Resch)_
+
+We continued work on Elektra's bindings for GNOME [GSettings](https://developer.gnome.org/gio/stable/GSettings.html). We aim to create a drop-in replacement for the [dconf](https://wiki.gnome.org/Projects/dconf) GSettings backend. _(Gabriel Rauter)_ and _(Mihael Pranjić)_
+
+### Bugfixes and Documentation Improvements
+
+This Elektra release brings many bugfixes and documentation improvements. We updated and improved the stability of our test infrastructure. Continue reading for the full list of improvements we have made.
```

Comment:
A sentence like this can stay, even if it is not a highlight. Note that there are also some new features, not everything is bugfixes&docu below. (e.g. `args=remaining` by @kodebach)

Reply:
Good point. I just wanted to highlight that, in contrast to what we did in previous releases, we now shifted the focus from new features to stability and KDE and GNOME integration.

Reply:
Then exactly write this, maybe in the beginning of KDE/GNOME, or even in the introductory text. It does not need to be a high-light.